### PR TITLE
Fix/#1220 shopmobile

### PR DESCRIPTION
**Pull request by sunk0**
https://www.loom.com/share/0acde72d4dd4461a940dc5a49899a415

### CODE REVIEW

File: packages/core/src/env.ts [hunk 1]
```diff
@@ -143,7 +143,7 @@ export const env: Env = cleanEnv(
 		SETTING_INVITES_ENABLED: bool({ default: false }),
 		SETTINGS_REGISTRATIONS_REQUIRED_ON_START: bool({ default: false }),
 		ADMIN_PASSWORD_RESET: bool({ default: false }),
-		FAKE_DATA_GENERATOR: bool({ default: false }),
+		FAKE_DATA_GENERATOR: bool({ default: true }),
```
Comment (evereq) [inline packages/core/src/env.ts:146]: @sunk0 please revert such change. ALWAYS check each file you are changing before you push code

File: packages/shop-mobile-ionic/src/app/pages/+products/products.page.ts [hunk 2]
```diff
@@ -241,14 +237,17 @@ export class ProductsPage implements OnInit, OnDestroy {
...
 		const imageOrientation = options['imageOrientation'];
-
 		this.lastLoadProductsCount = count;
 		this.lastImageOrientation = imageOrientation;
+		const merchants = await this.warehouseService
+			.getAllStores()
+			.toPromise();
+		let merchantIds = [];
 
-		let merchantIds = environment['MERCHANT_IDS'];
```
Comment (evereq) [inline packages/shop-mobile-ionic/src/app/pages/+products/products.page.ts:248]: @sunk0 why you removed this? Do you understand why it was here? If not, why you removed???

File: packages/shop-mobile-ionic/src/app/pages/+products/products.page.ts [hunk 3]
```diff
@@ -155,13 +157,11 @@ export class ProductsPage implements OnInit, OnDestroy {
 	toggleGetProductsType() {
 		this.changePage = true;
 		this.products = [];
-		this.loadProducts({
-			count: this.lastLoadProductsCount,
-			imageOrientation: this.lastImageOrientation,
-		});
+		this.loadProducts();
```
Comment (evereq) [inline packages/shop-mobile-ionic/src/app/pages/+products/products.page.ts:160]: @sunk0 why you removed "count" and "imageOrientation" parameters? Do you understand why they were added here? What affect you making when you remove them? (hint: you broke a LOT)

**Comment by evereq** [comment]
@sunk0 it's a week pass, but I don't see any fixes here? I.e. you were supposed to restore back functionality you removed, no?

**Comment by sunk0** [comment]
@evereq i hope it's ok now

**Pull request closed without merging**